server/src/test/java/org/spine3/server/entity/RecordBasedRepositoryShould.java

```diff
@@ -56,10 +59,10 @@
  */
 @SuppressWarnings("ConstantConditions")
 public abstract class RecordBasedRepositoryShould<E extends AbstractVersionableEntity<I, S>
-                                                          & TestEntityWithStringColumn,
-                                                  I,
-                                                  S extends Message>
-                                                  extends TenantAwareTest {
+        & TestEntityWithStringColumn,
```

Comment:
I would avoid `&` used for generics to discourage this type of design for Spine users.